go.mod

```
@@ -1,6 +1,6 @@
 module code.gitea.io/gitea
 
-go 1.24.6
+go 1.25.0
```

```suggestion
go 1.25.1
```

https://tip.golang.org/doc/devel/release#go1.25.1

---

fa0ef27